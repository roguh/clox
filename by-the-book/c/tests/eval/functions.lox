fun krabs(a,b){
    print("YES!"); print(a); print(b); return a * b;
}
print(krabs);
fun having() {
 yet();
}
fun yet() {
 print(krabs(22, 33));
}
having();

