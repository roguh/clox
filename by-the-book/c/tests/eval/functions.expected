<fn krabs>
YES!
22
33
726
